### PR TITLE
Feature/java17image

### PR DESCRIPTION
**Pull request by arunaukumar**
# Description

This pull requests includes upgradation of docker image to java 17 by keeping java 11 alive . 
# Checklist:

- [x] I have ensured the Defra version in the **JOB.env** file matches that in the **Dockerfile**
- [x] I have ensured the Java versions in the **image-matrix.json** match the **Dockerfile** and the table in the **README.md**
- [x] I have added newly ignored vulnerabilities to the **POLICY_CONFIGURATION.md**
- [x] I have checked if previously identified vulnerabilities have been patched, and can be removed from the **.grype.yaml** and **POLICY_CONFIGURATION.md** files

### CODE REVIEW

File: JOB.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 
-DEFRA_VERSION=1.1.1
-BASE_VERSION=11-jdk-jammy
+DEFRA_VERSION=1.1.2
+BASE_VERSION=17-jre-jammy
```
Comment (DaRoszko) [inline JOB.env:3]: should be JDK?

File: .github/workflows/build-scan-push.yml [hunk 2]
```diff
@@ -67,13 +67,14 @@ jobs:
...
           do
+            echo "github.ref  " ${{github.ref}}
             echo "creating tag ${{steps.setImageDetails.outputs.dockerRepoName}}:$TAG"
             docker image tag ${{steps.setImageDetails.outputs.fullImageName}} ${{steps.setImageDetails.outputs.dockerRepoName}}:$TAG
           done
 
       - name: Login to DockerHub
         uses: docker/login-action@v2
-        if: github.ref == 'refs/heads/main'
+       if: github.ref == 'refs/heads/main'
```
Comment (DaRoszko) [inline .github/workflows/build-scan-push.yml:77]: check indentation

File: JOB.env (hunk 1 above)
Comment (arunaukumar) [reply JOB.env]: fixed it

**Review by DaRoszko — APPROVED** [review]

**Review by ben-sagar — APPROVED** [review]
This looks good to me

**Pull request merged**